### PR TITLE
#25 Doc block adjustments, code formatting and added filter functiona…

### PR DESCRIPTION
**Pull request by linslin**
Simple way for filtering without using `preg_match` route patterns. Should be improved for url patterns soon. Also made code style conform to Yii2 guidelines, modified doc blocks and readme adjustments.

### CODE REVIEW

**Comment by linslin** [comment]
Spellings.

File: UrlManager.php [hunk 1]
```diff
@@ -90,6 +90,12 @@ class UrlManager extends BaseUrlManager
     public $languageParam = 'language';
 
     /**
+     * Array of filtered URL route strings
+     * @param array
```
Comment (phpnode) [inline UrlManager.php]: this is a `@var` rather than a `@param`

Comment (linslin) [inline UrlManager.php]: Yep, right. Sorry.

**Comment by mikehaertl** [comment]
@linslin Thanks, but before we do any implementation we should really first agree on how this feature will work. I'm still not sure about all the details and want to spend some more time with planning first.

**Comment by linslin** [comment]
Agree, meanwhile this feature is implemented on a productive system right now. Works like a charm for this moment. I totally agree to your thoughts. So, we shoud allow filter pattern's to make this work nice?

**Comment by linslin** [comment]
Any updates on this?

**Comment by mikehaertl** [comment]
Sorry, no, I had no time yet to look closer into this.

**Comment by mikehaertl** [comment]
@linslin I've used a slightly different approach, as your implementation was missing URL creation and used a too generic property name `filter`. So closing this for now. If you have feedback, please let me know in #25 .

**Pull request closed without merging**